commit/tokenprice/processor.go

```diff
@@ -85,6 +86,10 @@ func (p *processor) Outcome(
 ) (Outcome, error) {
 	lggr := logutil.WithContextValues(ctx, p.lggr)
 
+	if invalidateCache, ok := ctx.Value(consts.InvalidateCacheKey).(bool); ok && invalidateCache {
```

Comment:
Some comment on that invalidation? It's not obvious why it landed here. How can we test that to avoid someone accidentally removing it?